tutorial/20_recipes/012_artifact_tutorial.py

```diff
@@ -86,17 +86,17 @@
 
 First, we explain a simple case where the optimization is completed locally.
 
-Normally, Optuna's optimization history is persisted into some kind of a database via storage objects. Here, let's consider a 
-method using SQLite, a lightweight RDB management system, as the backend. With SQLite, data is stored in a single file (e.g., 
-./example.db). The optimization history comprises what parameters were sampled in each trial, what the evaluation values for those 
-parameters were, when each trial started and ended, etc. This file is in the SQLite format, and it is not suitable for storing 
-large data. Writing large data entries may cause performance degradation. Note that SQLite is not suitable for distributed parallel 
-optimization. If you want to perform that, please use MySQL as we will explain later, or JournalStorage 
+Normally, Optuna's optimization history is persisted into some kind of a database via storage objects. Here, let's consider a
+method using SQLite, a lightweight RDB management system, as the backend. With SQLite, data is stored in a single file (e.g.,
+``./example.db``). The optimization history comprises what parameters were sampled in each trial, what the evaluation values for those
+parameters were, when each trial started and ended, etc. This file is in the SQLite format, and it is not suitable for storing
+large data. Writing large data entries may cause performance degradation. Note that SQLite is not suitable for distributed parallel
+optimization. If you want to perform that, please use MySQL as we will explain later, or ``JournalStorage``
```

Comment:
How about using cross-reference?

```suggestion
optimization. If you want to perform that, please use MySQL as we will explain later, or :class:`~optuna.storages.JournalStorage`
```

Reply:
Thanks! I've committed your suggestion and remove the following line done as https://github.com/optuna/optuna/pull/5592/commits/30fbb4ad941a774b515a49ee9a98708e35d5bb43.